code/implementation/sample-setup/src/main/java/io/cattle/platform/sample/data/SampleDataStartupV10.java

```diff
@@ -29,6 +29,9 @@
     Configuration configuration;
```

Comment:
All changes to this file aren't needed per my comment https://github.com/rancher/cattle/pull/1994#issuecomment-248626932